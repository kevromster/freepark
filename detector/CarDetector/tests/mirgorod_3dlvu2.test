config = camera_nsk_mirgorod_3dlvu.txt
input = input/mirgorod_3dlvu2.png
expected = expected/mirgorod_3dlvu2.out
