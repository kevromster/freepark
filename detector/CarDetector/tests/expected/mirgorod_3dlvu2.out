[
parking id: 24
spots count: 3

{ lat: 54.853795; lon: 83.040075; complexity: NEWBIE }
{ lat: 54.853778; lon: 83.040087; complexity: NEWBIE }
{ lat: 54.853732; lon: 83.040121; complexity: NEWBIE }
]

[
parking id: 25
spots count: 1

{ lat: 54.853672; lon: 83.04017; complexity: NEWBIE }
]

[
parking id: 26
spots count: 3

{ lat: 54.8538; lon: 83.039992; complexity: EXPERIENCED }
{ lat: 54.853717; lon: 83.040068; complexity: NEWBIE }
{ lat: 54.853674; lon: 83.040106; complexity: NEWBIE }
]

